specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-03-01-preview/definitions.json

```diff
@@ -8761,6 +8761,73 @@
         "productId"
       ],
       "description": "Tag-product link entity properties."
+    },
+    "GatewayListDebugCredentialsContract": {
+      "type": "object",
+      "properties": {
+        "credentialsExpireAfter": {
+          "type": "string",
+          "format": "duration",
+          "description": "After what period of time to make credentials expire."
+        },
+        "purposes": {
+          "type": "array",
+          "description": "Purposes of debug credential.",
+          "items": {
+            "type": "string",
+            "description": "Purpose of debug credential.",
+            "enum": [
+              "tracing"
+            ],
+            "x-ms-enum": {
+              "name": "GatewayListDebugCredentialsContractPurpose",
+              "modelAsString": true,
+              "values": [
+                {
+                  "value": "tracing",
+                  "description": "The tracing purpose."
+                }
+              ]
+            }
+          }
+        },
+        "apiId": {
+          "type": "string",
+          "description": "Full resource Id of an API."
```

Comment:
can add format: arm-id and x-ms-arm-id-details
[https://azure.github.io/autorest/extensions/#x-ms-arm-id-details](https://azure.github.io/autorest/extensions/#x-ms-arm-id-details)

Reply:
Done